nipoppy_cli/nipoppy/config/main.py

```diff
@@ -31,6 +59,27 @@ class Config(ModelWithContainerConfig):
 
     model_config = ConfigDict(extra="allow")
 
+    def _check_sessions_have_prefix(self) -> Self:
+        """Check that sessions have the BIDS prefix."""
+        for session in self.SESSIONS:
+            check_session_strict(session)
+        return self
+
+    def _check_dicom_dir_options(self) -> Self:
+        """Check that only one DICOM directory mapping option is given."""
+        if (
+            self.DICOM_DIR_MAP_FILE is not None
+            and self.DICOM_DIR_PARTICIPANT_FIRST is not None
+        ):
+            raise ValueError(
+                "Cannot specify both DICOM_DIR_MAP_FILE and DICOM_DIR_PARTICIPANT_FIRST"
+            )
+        # otherwise set the default if needed
+        elif self.DICOM_DIR_PARTICIPANT_FIRST is None:
+            self.DICOM_DIR_PARTICIPANT_FIRST = True
+
+        return self
+
     def _check_no_duplicate_pipeline(self) -> Self:
```

Comment:
What about duplicates within `BIDS_PIPES` or `PROC_PIPES`?

Reply:
Right now that is not possible because BIDS pipelines have the `step` field. But will add that check after we change it (in #238)